### ISSUE TITLE
validationError is not respected with required

### ISSUE THREAD

**Issue by mitom** [zabute/formsy-semantic-ui-react#59]
Formsy provided `validationErrors` (object) and `validationError` (string) https://github.com/formsy/formsy-react/blob/master/API.md#validationerror The latter is used when one doesn't want specific error messages. This works when the validation error is from one of the specified `validators`, but doesn't when it comes from `required`.

As an example, the following should produce the same error message when the field is empty and when it's an invalid email.
```
              const errorLabel = <Label color='red' pointing />

              <Form.Input
                label='Email'
                type='email'
                name='username'
                validations='isEmail'
                errorLabel={errorLabel}
                validationError='Please enter valid email address.'
                required
              />
```

However it produces the below:

![image](https://user-images.githubusercontent.com/3754515/57392169-ff701480-71b7-11e9-986e-f442a554d25a.png)
![image](https://user-images.githubusercontent.com/3754515/57392177-026b0500-71b8-11e9-92df-a997d44ef3cd.png)

This is reproducible on the examples in the repo as well.

**Comment by zabute** [comment]
try this: 

```js
validationErrors={{
    isEmail: 'This is not a valid email',
    isDefaultRequiredValue: 'Email is Required',
 }}
```

**Comment by mitom** [comment]
Thanks @zabute, I know that `validationErrors`works. These 2 are for different purposes though. `validationError` serves the purpose of not having to provide an individual message for each validator, but can provide a simple instruction in case the validation fails _for any reason_, or a default one in case there is no explicitly specified one in `validationErrors` for the given error.

Furthermore, you can see from the above the case is not that the option is not present, it's that there is a bug somewhere causing it to not apply to `required`.

**Comment by zabute** [comment]
Yeah, Its a bug. Sorry for making you write your issue again - I skimmed through it the first time :(

**Comment by mitom** [comment]
No worries, thank you for your work!

**Comment by mitom** [comment]
It seems this is a bug in formsy itself, it was fixed a long time ago but they haven't made any releases since before then, so I suspect it will only be released in v2, which is still in beta. https://github.com/formsy/formsy-react/issues/123.

**Comment by rkuykendall** [comment]
This fix was released in Formsy ^2.0.0. It can be closed with https://github.com/zabute/formsy-semantic-ui-react/issues/65

**Comment by felixmosh** [comment]
Fix by #65
